dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task250107RemoveEsReadOnlyMonitorJob.java

```diff
@@ -0,0 +1,66 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+
+import java.util.ArrayList;
+
+public class Task250107RemoveEsReadOnlyMonitorJob implements StartupTask {
+
+
+    private final String JOB_NAME = "EsReadOnlyMonitorJob";
+    private final String TRIGGER_NAME = "trigger29";
+    private final String CRON_TABLE_NAME = "qrtz_excl_cron_triggers";
+    private final String TRIGGER_TABLE_NAME = "qrtz_excl_triggers";
+    private final String JOB_TABLE_NAME = "qrtz_excl_job_details";
+
+    /**
+     * Determines if the task should be forced to run.
+     *
+     * @return true if EsReadOnlyMonitorJob is found, false otherwise.
+     */
```

Comment:
I would encapsulate this in an Util at least that help us to remove any Job by rest or on demand